src/main/frontend/components/EditSubmittedTile.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+const EditSubmittedTile = props => {
+
+  return(
+    <div>
+      <h3>Your review edit has been submitted.</h3>
+      <button type="button" onClick={props.handleContinue}>Continue</button>
+    </div>
+  )
+}
+
+export default EditSubmittedTile;
```

Comment:
Is this component being imported and used anywhere? I might be missing it

Reply:
deleted this.